packages/react-app/src/components/modals/schedule-claim-modal.tsx

```diff
@@ -68,11 +68,20 @@ export default function ScheduleClaimModal({
   const formRef = useRef<HTMLFormElement>(null);
   const { setTransaction } = useTransactionContext();
   const modalId = useMemo(() => uniqueId('schedule-claim-modal'), []);
+  let mounted = true;
+
+  useEffect(
+    () => () => {
+      mounted = false;
+    },
+    [],
+  );
```

Comment:
Mayeb we can open PR to create some hook for it, i saw we have multiples those on the code base.

Reply:
I was thinking the same but wasnt able to have something cleaner